README.md

```diff
@@ -81,6 +81,44 @@ end
 
 Can be `http` or `https` protocol
 
+## Add your own service
+
+You can make available your own services:
+
+`my_awesome_service.rb`:
+
+```ruby
+module MarkdownVideos::Services
+
+  class MyAwsomeService < ServiceBase
+
+    def self.regexp
+      /(https?:\/\/myaweso.me\/video\/([\w\-]{4,16})\??([\w\-\=]+)?)/
+    end
+
+    def width
+      560
+    end
+
+    def height
+      315
+    end
+
+    def url
+      "https://player.myaweso.me/video/#{id}"
```

Comment:
If I didn't know how what ServiceBase looked like, I would be somewhat confused about this `id` method. I think you should probably define it in here too. Actually I would remove it from base altogether, and let each service be specific about it. You can add a `matches` (or `url_data`, whatever you want to call it) method in ServiceBase to make it easier.

```
def id
   url_data[2]
end
```